src/skills-builder/skills-builder-header/messages.js

```diff
@@ -0,0 +1,16 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  skillsBuilderHeaderTitle: {
+    id: 'skills.builder.header.title',
+    defaultMessage: 'Skills Builder',
+    description: 'Title for the Skills Builder feature',
+  },
+  skillsBuilderHeaderSubheading: {
+    id: 'skills.builder.header.subheading',
+    defaultMessage: 'Let edX be your guide',
```

Comment:
Hi - this pull request (specifically this line) introduced a bug:

https://github.com/openedx/frontend-app-profile/issues/686

Could you please address by refactoring the edX-specific language out of strings? Thank you.